lib/general_widgets/custom_user_channel.dart

```diff
@@ -1,45 +1,59 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:hng/ui/shared/styles.dart';
 
 class CustomUserChannel extends StatelessWidget {
   const CustomUserChannel(
-      {Key? key, this.image, this.text, this.trailingIcon, this.iconColor})
+      {Key? key, this.image, this.text, this.text2, this.trailingIcon, this.iconColor})
       : super(key: key);
   final image;
   final text;
+  final text2;
   final trailingIcon;
   final iconColor;
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-        height: 40,
-        width: 379,
-        child: Row(
-          children: [
-            Container(
-              child: Image(
-                height: 34,
-                width: 34,
-                fit: BoxFit.cover,
-                image: AssetImage(image),
-              ),
-            ),
-            SizedBox(width: 20),
-            Text(
-              text,
-              style: TextStyle(fontSize: 16, fontWeight: FontWeight.bold),
-            ),
-            SizedBox(width: 20),
-            Container(
-              height: 7,
-              width: 7,
-              child: Icon(
-                trailingIcon,
-                size: 9,
-                color: iconColor,
-              ),
-            )
-          ],
-        ));
+    return ScreenUtilInit(
+      designSize: const Size(411, 823),
+      builder: () => InkWell(
+        onTap: () {},
+        child: Container(
+            height: 24.h,
+            width: 211.w,
+            child: Row(
+              children: [
+                Container(
+                  child: Image(
+                    height: 24.h,
+                    width: 24.w,
+                    fit: BoxFit.cover,
+                    image: AssetImage(image),
+                  ),
+                ),
+                SizedBox(width: 12.w),
+                Text(
+                  text,
```

Comment:
textData here can be null and twill crash the app. Please set a default  value, return an empty string if null or make its field  "required"

Reply:
done, and its just a template for a user coming from the endpoints,